services/virtual-assistant/src/virtual_assistant/assistant/watson.py

```diff
@@ -161,7 +162,9 @@ async def create_session(self, user_id: str) -> str:
         )
         return response.get_result()["session_id"]
 
-    async def send_message(self, message: AssistantInput) -> AssistantOutput:
+    async def send_message(
+        self, message: AssistantInput, context: dict
```

Comment:
not really convinced about this approach of making it a `dict`.

Lets make a dataclass with the data that we care about, `AssistantContext` or `Context` maybe? 

```python
class Context:
  is_internal: boolean
  is_org_admin: boolean
```

Each assistant will decide how it gets sent to the service.

Reply:
Thank you, made a lot more sense. I didn't want the talk endpoint to depend on the watson file, so I moved the vars outside to that other skills.py file haha

Adding that struct to the Assistant package worked a lot better :+1: